Mathlib/Analysis/Calculus/FDeriv/Analytic.lean

```diff
@@ -361,6 +580,30 @@ protected theorem hasFDerivAt [DecidableEq ι] : HasFDerivAt f (f.linearDeriv x)
   convert f.hasFiniteFPowerSeriesOnBall.hasFDerivAt (y := x) ENNReal.coe_lt_top
   rw [zero_add]
 
+/-- Given `f` a multilinear map, then the derivative of `x ↦ f (g₁ x, ..., gₙ x)` at `x` applied
+to a vector `v` is given by `∑ i, f (g₁ x, ..., g'ᵢ v, ..., gₙ x)`. Version inside a set. -/
+theorem _root_.HasFDerivWithinAt.multilinear_comp
+    [DecidableEq ι] {G : Type*} [NormedAddCommGroup G] [NormedSpace 𝕜 G]
```

Comment:
Do you need `DecidableEq` to state the theorem? If no, then please use `classical` in the proof instead.

Reply:
I see that `ContinuousMultilinearMap.toContinuousLinearMap` uses `DecidableEq`.